http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpAppDAO.scala

```diff
@@ -14,18 +14,32 @@ class HttpAppDAO[F[_]: Async](val kubernetesDnsCache: KubernetesDnsCache[F], cli
 
   def isProxyAvailable(googleProject: GoogleProject, appName: AppName, serviceName: ServiceName): F[Boolean] =
     Proxy.getAppTargetHost[F](kubernetesDnsCache, CloudContext.Gcp(googleProject), appName) flatMap {
-      case HostReady(targetHost, _, _) => // Update once we support Relay for apps
-        client
-          .successful(
-            Request[F](
-              method = Method.GET,
-              uri = Uri.unsafeFromString(
-                s"https://${targetHost.address}/proxy/google/v1/apps/${googleProject.value}/${appName.value}/${serviceName.value}/"
+      case HostReady(targetHost, _, _) =>
+        serviceName match {
+          case ServiceName("welder-service") =>
```

Comment:
Maybe this could be DRY'd a little bit since only the URL is different?
```
case HostReader(targetHost, _, _) =>
  val url = serviceName match {
    case ServiceName("welder") => "…/status"
    case _ => …
  }
  client.successful(…)
}
```

Reply:
Also in the back of my mind is restructuring code so adding new apps/services don't entail adding a million new `case` statements… but that's out of scope of this PR I think.

Reply:
That makes a lot of sense, I need to step up my scala syntax a bit, thanks!